code/modules/assembly/flash.dm

```diff
@@ -168,7 +182,7 @@
 				if(M.stat == CONSCIOUS)
 					M.mind_initialize() //give them a mind datum if they don't have one.
 					var/resisted
-					if(!isloyal(M) && !jobban_isbanned(M, "catban"))
```

Comment:
oh shit I didn't even notice that one, it doesn't even check the loyalty implants fuck
fixing